experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -334,6 +350,99 @@ export class NodeSDK {
     );
   }
 
+  public configureMetricProviderFromEnv(): MetricReader[] {
```

Comment:
This doesn't need to be a method on the class because it isn't accessing `this`. Making it a function improves minification and tree-shaking. It looks like the function wouldn't need to be exported too, keeping the external API surface as small as possible. It should at least not be public.

Reply:
done

Reply:
Done